### PR TITLE
Update git-latexdiff

### PR DESCRIPTION
**Pull request by rubendibattista**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

MacOS 10.13.5 17F77
Xcode 9.4.1 9F2000

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/git-latexdiff/Portfile [hunk 1]
```diff
@@ -18,11 +18,12 @@ long_description    Tool to graphically visualize differences between \
...
-master_sites        https://gitlab.com/${name}/${name}/repository/archive.tar.gz?ref=v${version}&dummy=
-distname            ${name}-v${version}-2bd0dcbf2a14de7c311382ed03eb24bac661ac69
+master_sites        https://gitlab.com/${name}/${name}/-/archive/v${version}
+distname            ${name}-v${version}
 
-checksums           rmd160  29b61b526bbed34d32ed8d945c3e569a36d10fc7 \
-                    sha256  76b19b623749025bde0617776237dbd6763a21d19cd6b6b353765846dac766ec
+
+checksums           rmd160  9e89a7973114af83daff586b76740f44912b71df \
+                    sha256  54ba32ca75feef37a8d47b7b35d29a82819031a408bdb3eb25c1da73e68811e4
```
Comment (pmetzger) [inline devel/git-latexdiff/Portfile]: Please add a `size` field to `checksums` and we will merge this.

Comment (rubendibattista) [reply devel/git-latexdiff/Portfile]: Where do I find an example or documentation on the `size` field?

Comment (mf2k) [reply devel/git-latexdiff/Portfile]: It's not documented yet. But you can look at another port with it such as py-numpy. You can determine the correct value by initially setting it to 1 and then run `sudo port -v checksum git-latexdiff` to get the correct value.

Comment (pmetzger) [reply devel/git-latexdiff/Portfile]: @mf2k I was unaware it wasn't currently in the guide. Could you add it? I'd do it but I've got limited time ATM.

File: devel/git-latexdiff/Portfile [hunk 2]
```diff
@@ -18,11 +18,13 @@ long_description    Tool to graphically visualize differences between \
...
 
 homepage            https://gitlab.com/${name}/${name}
-master_sites        https://gitlab.com/${name}/${name}/repository/archive.tar.gz?ref=v${version}&dummy=
-distname            ${name}-v${version}-2bd0dcbf2a14de7c311382ed03eb24bac661ac69
+master_sites        https://gitlab.com/${name}/${name}/-/archive/v${version}
+distname            ${name}-v${version}
 
-checksums           rmd160  29b61b526bbed34d32ed8d945c3e569a36d10fc7 \
-                    sha256  76b19b623749025bde0617776237dbd6763a21d19cd6b6b353765846dac766ec
+
```
Comment (pmetzger) [inline devel/git-latexdiff/Portfile]: The file didn't need this extra newline, but I'm not going to hold up committing it for this. If you update again in the future, get rid of it. 😀

**Comment by pmetzger** [comment]
Merged. Thank you, @rubendibattista!

**Comment by jjstickel** [comment]
I was on vacation last week and didn't have a chance to look at this until now. Looks good. Thanks for the update!

**Pull request merged**